### DISCUSSION TITLE
Navigate to Newly Created Thing

### DISCUSSION THREAD

**Discussion by dw-bayer** [Budibase/budibase#3357] [Help] [answered]
When you save a Create type Form is there a way to get that id?
In the Save action I'd like to navigate to the details page instead of going back to the list page.

**Comment by oswaldB** [comment, marked as answer]
Hello,

As far as I know it is not possible.

But you can lead your user on page where your data provider has a filter on the user created data, sort on the created data and you limit to one.

It might work

**Comment by shogunpurple** [comment]
@oswaldB is correct - the row ID is not available as a binding for a newly created item, so you cannot currently do this.

I'm going to convert this to an issue, and we will treat it as a feature request because it would be quite useful to have. The team will look into it. Thanks!
